msal/src/Microsoft.Identity.Client/Exceptions/MsalError.cs

```diff
@@ -172,5 +172,10 @@ internal static class MsalError
         /// An error response was returned by the OAuth2 server and it could not be parsed
         /// </summary>
         public const string NonParsableOAuthError = "non_parsable_oauth_error";
+
+        /// <summary>
+        /// Device code expired before contacting the server
```

Comment:
In the context of Device code flow (See https://aka.ms/msal-net-device-code-flow), this error happens when the device code expired before the user signed-in on another device (this is usually after 15 mins)